src/proteus/dswx_hls.py

```diff
@@ -3435,12 +3497,20 @@ class in the browse image. False to display PSW-Agg.
     logger.info(f'    product version: {product_version}')
     logger.info(f'    software version: {SOFTWARE_VERSION}')
     logger.info(f'processing parameters:')
-    logger.info(f'    flag_use_otsu_terrain_masking: '
-                                f'{flag_use_otsu_terrain_masking}')
-    logger.info(f'    min_slope_angle: {min_slope_angle}')
-    logger.info(f'    max_sun_local_inc_angle: {max_sun_local_inc_angle}')
-    logger.info(f'    mask_adjacent_to_cloud_mode: '
-                                f'{mask_adjacent_to_cloud_mode}')
+    if flag_use_otsu_terrain_masking:
+        terrain_masking_algorighm = 'Otsu'
+        terrain_masking_parameters_str = ' (unused)'
+    else:
+        terrain_masking_algorighm = 'Sun local incidence angle'
+        terrain_masking_parameters_str = ''
+
+    logger.info(f'    terrain masking algorithm: {terrain_masking_algorighm}')
```

Comment:
This is clearer now, thank you! A couple of quick suggestions: First, there is a small typo in the variable name. Second, could we still log the output of the actual runconfig parameter supplied? What do you think about something like this?

```suggestion
        terrain_masking_algorithm = 'Otsu'
        terrain_masking_parameters_str = ' (unused)'
    else:
        terrain_masking_algorithm = 'Sun local incidence angle'
        terrain_masking_parameters_str = ''

    logger.info(f'    flag_use_otsu_terrain_masking: {flag_use_otsu_terrain_masking}')
    logger.info(f'    terrain masking algorithm: {terrain_masking_algorithm}')
```

Reply:
It sounds redundant to me, @nemo794 . We are already printing unused parameters (for Otsu), not sure why we want to print two lines with the same info?

Reply:
I do not think they are redundant. `flag_use_otsu_terrain_masking` tells what the input runconfig parameter was, and `terrain_masking_algorithm` tells in human-friendly words which algorithm is being used.

Also, if I'm the user and I'm seeing the log say: `terrain masking algorithm:  Sun local incidence angle`, then I might not know where that is coming from, nor how/where to modify it. By seeing the _actual_ runconfig parameter name `flag_use_otsu_terrain_masking`, then I might have a hint as to where to go to modify this field.

Reply:
Well, I still think it would be redundant to keep both messages. To make everyone happy, I added enums to the runconfig and now I'm printing them out.